### ISSUE TITLE
pxlssq gets Nexus labels wrong if contiain spaces

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#154]
Related to fbfef0d. When read in by `pxlssq`, the label is terminated at the first space (token). Can be seen with:

```
pxlssq -s seq_file -l
```

Example:
```
'GU931859.1 Dendroica adelaidae voucher STRI-SLDAD10 aconitase 1 (ACO1) gene, exon 8 and partial cds'
```
is reported as:
```
GU931859.1
```
It also obviously messes up character state counts, number of characters, whether it is aligned, etc. Oof.

TODO: if label begins with a quote, read label until closing quote. This can get more complicated when the label itself contains quotes (e.g., "Bewick's Wren"). Don't worry about that until it surfaces.

**Comment by josephwb** [comment]
Involves both `read_next_seq_from_stream()` and `read_interleaved_nexus()`.

**Comment by josephwb** [comment]
First one fixed with 496bea7

**Comment by josephwb** [comment]
And the second bf78f55
